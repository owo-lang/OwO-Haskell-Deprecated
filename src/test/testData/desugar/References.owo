module References where

 A = Type

 B : Type1
 B = A
