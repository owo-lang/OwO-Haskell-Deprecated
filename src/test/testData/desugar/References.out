File type: owo
Top module name: .References
 implementation of function A (26 3 2) (27 3 3)
 type
  meta variable _ (26 3 2) (27 3 3)
 value
  type universe Type (30 3 6) (34 3 10)
  level: ULevelLit 0
 implementation of function B (48 6 2) (49 6 3)
 type
  type universe Type1 (41 5 6) (46 5 11)
  level: ULevelLit 1
 value
  global reference A (52 6 6) (53 6 7)
